examples/poyo/train.py

```diff
@@ -0,0 +1,362 @@
+import logging
+from typing import Callable, Dict
+import copy
+
+import hydra
+import lightning as L
+import torch
+import torch.nn as nn
+from torch.utils.data import DataLoader
+from torch_optimizer import Lamb
+from lightning.pytorch.callbacks import (
+    LearningRateMonitor,
+    ModelCheckpoint,
+    ModelSummary,
+)
+from omegaconf import DictConfig, OmegaConf
+from temporaldata import Data
+
+from torch_brain.registry import MODALITIY_REGISTRY, ModalitySpec
+from torch_brain.models.poyo import POYOTokenizer, poyo_mp
+from torch_brain.utils import callbacks as tbrain_callbacks
+from torch_brain.utils import seed_everything
+from torch_brain.utils.stitcher import DecodingStitchEvaluator
+from torch_brain.data import Dataset, collate
+from torch_brain.data.sampler import (
+    DistributedStitchingFixedWindowSampler,
+    RandomFixedWindowSampler,
+)
+from torch_brain.transforms import Compose
+
+# higher speed on machines with tensor cores
+torch.set_float32_matmul_precision("medium")
+
+logger = logging.getLogger(__name__)
+
+
+@hydra.main(version_base="1.3", config_path="./configs", config_name="train.yaml")
+def main(cfg: DictConfig):
```

Comment:
the python convention is that `main()` goes at the bottom of the script. so this should be the last function.

Reply:
yeah we can rename this to `train()` or smth. Otherwise I don't like having both DataModule and TrainWrapper at the top. Too much code before you get to what this script is actually running. Happy to reconsider.

Reply:
let's put it back the way it was please. if you want to make the change across the board, it should be a separate PR/ a separate discussion.